src/ascent/ascent_jupyter_bridge/README.md

```diff
@@ -18,12 +18,24 @@ Bridge Kernel is not on PyPI yet, please clone and install from local source.
 
 spack/bin/spack install -v py-pip ^python@3.6.3
 spack/bin/spack activate py-pip
-pip install ipykernel
+pip install -r requirements.txt
 pip install . [--user] [--upgrade]
```

Comment:
we may want to add a note about how to persevere in the face of SSL cert errors.

`pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements.txt`